src/core_plugins/inspector_views/public/requests/_requests.scss

```diff
@@ -18,3 +18,11 @@
 .insRequestSelector__menuSpinner {
   margin-left: $euiSizeS;
 }
+
+.insRequestDetailsRequest__parent {
+  position: relative; }
+
+.insRequestDetailsRequest__devToolButton {
+  position: absolute;
+  z-index: 10;
+  right: 6px; }
```

Comment:
Class names look good, but try to use EUI Sass variables where you can. For instance, change `right: 6px` to `right: $euiSizeS` and you only need to bump the z-index to `1`.

Also, watch your closing brackets. They should be on their own line.